README.md

```diff
@@ -1,2 +1,19 @@
-# handbook-of-best-practices
-Software Quality Assurance Assignment
+## Code Reviews
+
+
+Code reviews are a very important part of the software development process which involves analysing
+the source code to identify bugs. A code review process is typically conducted 
+before merging with the codebase.
+
+Code reviews improve code quality making your codebase more stable. They prevent bugs and errors
+from getting into your project at an early stage of the software development process. 
```

Comment:
Only one suggestion maybe introduce the points that will be covered in the code review section of the handbook.

Reply:
Yeah, I'll do that now thanks!